.docsettings.yml

```diff
@@ -51,7 +51,8 @@ known_content_issues:
   - ['sdk/storage/azure-storage-file-share/swagger/README.md', '#4554']
   - ['sdk/storage/azure-storage-queue/swagger/README.md', '#4554']
   - ['sdk/storage/azure-storage-blob/swagger/README.md', '#4554']
-
+  - ['sdk/storage/azure-storage-file-datalake/swagger/README.md', '#4554']
+  -
```

Comment:
Remove this extra minus sign and Doc Warden should be happier. @scbedd  (saw an error here: https://dev.azure.com/azure-sdk/public/_build/results?buildId=181603&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=9069f0a2-bd31-51b1-8b5c-08355cf80229&l=84)

Reply:
Just previously there was also a minus sign, it didn't fail the readme test, weird 😄